orangecontrib/network/widgets/OWNxTwitterGraph.py

```diff
@@ -0,0 +1,248 @@
+import twitter
+import networkx as nx
+from Orange.widgets import widget, gui
+from Orange.data import Domain, StringVariable, Table
+import numpy as np
+from Orange.widgets.credentials import CredentialManager
+import orangecontrib.network as network
+from orangecontrib.text import twitter as txt_twitter
```

Comment:
This would probably fail if the user doesn't have Text add-on installed, too. Could/should we re-implement this here as well?